backend/src/test/java/corea/review/service/ReviewServiceTest.java

```diff
@@ -0,0 +1,66 @@
+package corea.review.service;
+
+import config.ServiceTest;
+import corea.exception.CoreaException;
+import corea.fixture.MatchResultFixture;
+import corea.fixture.MemberFixture;
+import corea.fixture.RoomFixture;
+import corea.matching.domain.MatchResult;
+import corea.matching.domain.ReviewStatus;
+import corea.matching.repository.MatchResultRepository;
+import corea.member.domain.Member;
+import corea.member.repository.MemberRepository;
+import corea.room.domain.Room;
+import corea.room.repository.RoomRepository;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.transaction.annotation.Transactional;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+@ServiceTest
+class ReviewServiceTest {
+    @Autowired
+    MemberRepository memberRepository;
+    @Autowired
+    RoomRepository roomRepository;
+    @Autowired
+    ReviewService sut;
+    @Autowired
+    MatchResultRepository matchResultRepository;
```

Comment:
[질문] `ReviewService`의 변수명이 sut인 이유가 뭘까요?

```suggestion
class ReviewServiceTest {

    @Autowired
    MemberRepository memberRepository;
    
    @Autowired
    RoomRepository roomRepository;
    
    @Autowired
    ReviewService sut;
    
    @Autowired
    MatchResultRepository matchResultRepository;
```

Reply:
`private` 붙이세요 🔪

Reply:
>[질문] ReviewService의 변수명이 sut인 이유가 뭘까요?

저도 sut 보단 단순히 `reviewService`처럼 풀어놓는게 좋다 생각됩니당~